doc/code/Doxyfile

```diff
@@ -755,8 +755,7 @@ WARN_LOGFILE           =
 
 INPUT                  = . \
                          @CMAKE_CURRENT_SOURCE_DIR@/applications \
```

Comment:
"applications" only contains python in sam branch. So should be removed when merged.